source/content/tax-exempt-status.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Request Tax Exempt Status
+description: Provide this information to file your qualified tax exempt status with Pantheon.
+tags: [billing]
+categories: [manage]
+contributors: [edwardangert]
+searchboost: 150
+reviewed: "2020-03-24"
+---
+
+To make it easy for our team to confirm your tax-exempt status and have it reflected on your bill quickly, [contact Support](/support) with the following information:
+
+- Copy of tax exemption determination letter
+- Email address of Site Owner
```

Comment:
Does this mean you need to provide the email address of the site owner (i.e. know it), or be contacting support _from_ a user associated with that email?

Reply:
attempted to clarify in L11